### PR TITLE
Change Roboto to `system-ui`

### PR DESCRIPTION
**Pull request by danbee**
This means that we're no longer loading a web font as the default font. This reduces remote dependencies and overall bandwidth of a page.

Different platforms will get slightly different fonts, with iOS and macOS getting San Francisco, Windows getting Segue UI, and Android getting Roboto. These fonts fit the platforms better and have similar metrics and appearance to Roboto.

This also means that if a site loads a custom font, they're only loading the custom font and not Roboto as well.

### CODE REVIEW

**Review by jadefish — COMMENTED** [review]
This is a huge diff for a small change – did something with bundle generation change?

**Comment by danbee** [comment]
@jadefish Yeah, it looks like the way the output CSS is formatted has changed. This should probably be rolled into the SCSS build PR.

File: views/mdc/assets/scss/material.blue_grey-orange.min.css
Comment (jadefish) [inline views/mdc/assets/scss/material.blue_grey-orange.min.css:1]: this looks like it got unminified (... maxified?), too. what touched this file?

Comment (jadefish) [reply views/mdc/assets/scss/material.blue_grey-orange.min.css:1]: ah, nvm, it's the default theme (I think?) – I read the file name wrong. this makes sense!

**Review by jadefish — APPROVED** [review]

**Pull request merged**